src/classes/isotopologue.cpp

```diff
@@ -81,3 +81,11 @@ Sears91::Isotope Isotopologue::atomTypeIsotope(std::shared_ptr<AtomType> at) con
 
 // Return AtomType/Isotope pairs list
 const std::vector<std::tuple<std::shared_ptr<AtomType>, Sears91::Isotope>> &Isotopologue::isotopes() const { return isotopes_; }
+
+toml::basic_value<toml::discard_comments, std::map, std::vector> Isotopologue::serialize()
+{
+    toml::basic_value<toml::discard_comments, std::map, std::vector> isotopologue;
+    for (auto &isotope : isotopes_)
+        isotopologue[std::get<0>(isotope).get()->name().data()] = Sears91::A(std::get<1>(isotope));
```

Comment:
```suggestion
    for (auto &[type, isotope] : isotopes_)
        isotopologue[type->name().data()] = Sears91::A(isotope);
```

Structured binding for clearer code.